### DISCUSSION TITLE
Issue 中的 QQ群链接失效

### DISCUSSION THREAD

**Discussion by AntoineChn** [zotero-chinese/wiki#210] [求助 / Q&A] [answered]
在 Issue 页面（https://github.com/zotero-chinese/wiki/issues/new/choose），选择「QQ群」按钮时，无法正确获取 QQ 群；并且，浏览器自动跳转到 https://zotero-chinese.com/plugins/。

**Comment by AntoineChn** [comment]
找到 QQ群号了，在 https://zotero-chinese.com/plugins/ 页面底部。

**Comment by northword** [comment, marked as answer]
已将该地址替换为 参与指南：https://zotero-chinese.com/contributing/#%E5%8F%82%E4%B8%8E%E8%AE%A8%E8%AE%BA
